collectors/osidb_scheduler/collectors.py

```diff
@@ -0,0 +1,73 @@
+"""
+OSIDB Collectors
+"""
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import OuterRef, Q, Subquery
+from django.db.models.functions import Cast
+
+from collectors.framework.models import Collector
+from osidb.core import set_user_acls
+from osidb.mixins import Alert
+
+logger = get_task_logger(__name__)
+
+
+class StaleAlertCollector(Collector):
```

Comment:
The collector framework is too have for this periodic job. It was designed for the data collection so the model has attributes like collected data status, up until when the data is complete, etc. while nothing like this is necessary here. You just perform full cleanup every time this job runs and that is it. No information to store needed, no status etc. Just a simple periodic Celery task should be enough or not? We do not have any framework for simple internal tasks like this (to eg. report status on API if needed etc.) but also we currently do not have a need for one :smile: so I would think that Celery should be enough.